src/main/java/de/ait/secondlife/controllers/CategoryController.java

```diff
@@ -84,6 +87,7 @@ public ResponseEntity<List<CategoryDto>> getAllPlusHidden() {
                     content = @Content(mediaType = "application/json", schema = @Schema(implementation = ResponseMessageDto.class)))}
     )
     public ResponseEntity<CategoryDto> add(@Valid @RequestBody CategoryCreationDto dto) {
+        utilities.checkCredentialsToConnectImageToEntity(dto.getBaseNameOfImages());
```

Comment:
I would move all these checkings to some service class. First, this is a business logic. Second, for checking you use another service classes

Reply:
fixed